rebasehelper/tests/public_api/test_specfile.py

```diff
@@ -48,6 +49,18 @@ def test_update_changelog(self, spec_object):
         assert spec_object.update_changelog('test2') is None
         assert spec_object.update_changelog(changelog_entry='test') is None
 
+    def test_get_new_log_with_non_c_locale(self, spec_object):
+        # Changelogs should be identical no matter what locale
+        changelog = spec_object.get_new_log("test2")
+        for l in locale.locale_alias:
```

Comment:
There is no other locale but C in Fedora container images, so this doesn't actually test anything when run on Travis for example.

Reply:
Does this mean we should add another locale to the images to be used in Travis testing? Or that we're fine with this test being a noop in Travis?

Reply:
I think it would make sense to add at least one (non-English) locale, but let me take care of that.

Reply:
Thank you.